### PR TITLE
use byte_range when reading CSVs

### PR DESCRIPTION
**Pull request by quasiben**
This PR fixes issue #71 .  There may be a small issue with cudf (https://github.com/rapidsai/cudf/issues/928) .  Once that issue is resolved (perhaps user error on my part) test should pass and this will be merged

### CODE REVIEW

File: dask_cudf/io/csv.py [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
 from glob import glob
 
 import cudf
 from dask.base import tokenize
 from dask.compatibility import apply
 import dask.dataframe as dd
 
-
-def read_csv(path, **kwargs):
+def read_csv(path, chunksize=25e6, **kwargs):
```
Comment (mrocklin) [inline dask_cudf/io/csv.py]: I recommend two things:

1.  Use `dask.utils.parse_bytes` on chunksize so users can providev values like `"25 MiB"`
2.  Use a larger blocksize.  I'll suggest "128 MiB", but @kkraus14 may have other suggestions

File: dask_cudf/io/tests/test_csv.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import dask_cudf
 import dask.dataframe as dd
 
+dask.config.set(scheduler='single-threaded')
```
Comment (mrocklin) [inline dask_cudf/io/tests/test_csv.py]: We can probably remove this

File: dask_cudf/io/csv.py (hunk 1 above)
Comment (kkraus14) [reply dask_cudf/io/csv.py]: Typically the larger the better, but "128 MiB" sounds like a good starting point.

File: dask_cudf/io/tests/test_csv.py [hunk 3]
```diff
@@ -11,3 +10,13 @@ def test_read_csv(tmp_path):
...
     df2 = dask_cudf.read_csv(tmp_path / "*.csv")
     dd.assert_eq(df, df2)
+
+def test_read_csv_w_bytes(tmp_path):
+    df = dask.datasets.timeseries(dtypes={"x": int, "y": int}, freq="120s").reset_index(
+        drop=True
+    )
+    df.to_csv(tmp_path / "data-*.csv", index=False)
+
+    df2 = dask_cudf.read_csv(tmp_path / "*.csv", chunksize='256 MiB')
```
Comment (mrocklin) [inline dask_cudf/io/tests/test_csv.py]:
```suggestion
    df2 = dask_cudf.read_csv(tmp_path / "*.csv", chunksize='1 kiB')
    assert df2.npartitions > df.npartitions
```

File: dask_cudf/io/tests/test_csv.py [hunk 4]
```diff
@@ -11,3 +11,14 @@ def test_read_csv(tmp_path):
...
     dd.assert_eq(df, df2)
+
+
+def test_read_csv_w_bytes(tmp_path):
+    df = dask.datasets.timeseries(dtypes={"x": int, "y": int}, freq="120s").reset_index(
+        drop=True
+    )
+    df.to_csv(tmp_path / "data-*.csv", index=False)
+
+    df2 = dask_cudf.read_csv(tmp_path / "*.csv", chunksize="1 kiB")
```
Comment (mrocklin) [inline dask_cudf/io/tests/test_csv.py]:
```suggestion
    df2 = dask_cudf.read_csv(tmp_path / "*.csv", chunksize="1 kiB")
    assert df2.npartitions > df.npartitions
```

**Pull request merged**